les/peer.go

```diff
@@ -423,11 +423,11 @@ func (p *peer) Handshake(td *big.Int, head common.Hash, headNum uint64, genesis
 		p.fcCosts = list.decode()
 	} else {
 		//on client node
-		p.announceType = announceTypeSimple
+		p.requestAnnounceType = announceTypeSimple
 		if p.isTrusted {
-			p.announceType = uint64(announceTypeSigned)
+			p.requestAnnounceType = uint64(announceTypeSigned)
```

Comment:
Question here: Why are the `requestAnnounceType` and the `announceTypeSimple` of type `uint64` while the `announceTypeSigned` has to be casted?